content/fr/docs/reference/glossary/admission-controller.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Admission Controller
+id: admission-controller
+date: 2019-06-28
+full_link: fr/docs/reference/access-authn-authz/admission-controllers/
+short_description: >
+  Un morceau de code qui intercepte les requêtes adressées au serveur API de Kubernetes avant la persistance de l'objet.
+
+aka:
+tags:
+- extension
+- security
+---
+Un morceau de code qui intercepte les requêtes adressées au serveur API de Kubernetes avant la persistance de l'objet.
+
+<!--more-->
+
+Les contrôleurs d'accès sont configurables pour le serveur API de Kubernetes. Un contrôleur d'accès peut être "validant"  ("validating"), "mutant" ("mutating"), ou
```

Comment:
```suggestion
Les contrôleurs d'accès sont configurables pour le serveur API de Kubernetes. Un contrôleur d'accès peut être "validant"  ("validating"), "changant" ("mutating"), ou
```
Complex to find a translation to this word

Reply:
I also considered "modifiant". How does that sound ? @perriea

Reply:
Perfect !

Reply:
Modifiant c'est bien

Reply:
@remyleone @perriea "Modifiant" it is then! Updated.